v22.2/transactions.md

```diff
@@ -128,6 +128,8 @@ transaction to retry, the client needs to be involved in retrying the whole
 transaction and so you should write your transactions to use
 [client-side intervention](#client-side-intervention).
 
+**New in v22.2:** The [`enable_implicit_transaction_for_batch_statements` session variable](#set-vars) defaults to `true`. This means that any batch of statements is treated as an implicit transaction, so the `BEGIN`/`COMMIT` commands are not needed to group all the statements in one transaction.
```

Comment:
I think to get the build green this link to `#set-vars` needs to be updated to point to `set-vars.html#enable-implicit-transaction-for-batch-statements`